openmeetings-server/src/site/xdoc/BuildInstructions.xml

```diff
@@ -60,7 +60,7 @@
 		<section name="Check dependencies">
 			<source>mvn org.apache.maven.plugins:maven-dependency-plugin:analyze-only</source>
 		</section>
-		<section name="Tips and Gotchas">
+		<section name="Incremental Build and Run Embedded Jetty">
```

Comment:
I only have concern regarding this section renaming: only 1 item is about Jetty ...
See no reason to change the section name :)

Reply:
It actually mentions in this section:
mvn clean -P allModules,quick,mysql jetty:run-exploded

Which will run the Embedded Jetty

Reply:
this section contains 5-6 run commands
and only one of them is about Jetty
others are about Tomcat ....

Reply:
yeah the other commands are all about incremental building modules

Reply:
well I rename it back then